.github/workflows/scheduled.yml

```
@@ -124,6 +124,12 @@ jobs:
           brew install curl
           echo "HOMEBREW_FORCE_BREWED_CURL=1" >>"${GITHUB_ENV}"
 
+      - name: Install and use Homebrew svn if needed
+        run: |
+          if brew ruby -e 'puts Formula[ARGV.first].deps.any? { |d| d.name == "subversion" && d.implicit? }' "$FORMULA"; then
```

```suggestion
          if [[ "$(brew ruby -e 'puts Formula[ARGV.first].deps.any? { |d| d.name == "subversion" && d.implicit? }' "$FORMULA")" = "true" ]]; then
```

Since the `brew ruby -e 'puts ...'` command is expected to always succeed regardless of its output.

---

Could also do
```suggestion
          if brew ruby -e 'exit 1 if Formula[ARGV.first].deps.none? { |d| d.name == "subversion" && d.implicit? }' "$FORMULA"; then
```